bundle-workflow/src/test_workflow/put_file_into_S3.py

```diff
@@ -0,0 +1,361 @@
+# !/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+import argparse
+import json
+import os
+
+from src.read_write_s3 import S3Bucket
+
+
+class ConvertWrite:
+    """
+    result_file- the file where the console output will be stored
+    assume_role- Amazon Resource Name (ARN) of the role that you want to assume
+    session- Name of the session
+    bucket_name- Name of the Bucket
+    object_path- Path in the Bucket where you want to store the file
+    file_path- File path where the console output result_file is present
+    """
+    def parse_arguments(self):
+        parser = argparse.ArgumentParser()
+        parser.add_argument("-r", "--result_file", help="Result file", required=True)
+        parser.add_argument("-a", "--assume_role", nargs="?", default="os.getenv('OPENSEARCH_TESTING_ROLE')", help="Assume role")
+        parser.add_argument("-s", "--session", nargs="?", default="get-files-session", help="Session name")
+        parser.add_argument("-b", "--bucket_name", help="Bucket name", required=True)
+        parser.add_argument("-o", "--object_path", help="ObjectPath in Bucket", required=True)
+        parser.add_argument("-f", "--file_path", help="Filepath of the result file", required=True)
+
+        args = parser.parse_args()
+
+        return args
+
+    def conversion_to_html(self, args):
+        with open(args.file_path + args.result_file) as json_data:
+            try:
+                data = json.load(json_data)
+                testExecutionId = data['testExecutionId']
+                filename = "%s.html" % testExecutionId
+                values = {1: 'opCount', 2: 'opErrorCount', 3: 'opErrorRate', 4: 'p0', 5: 'p50', 6: 'p100', 7: 'p50',
+                          8: 'p90',
+                          9: 'p99', 10: 'p100'}
+
+                stored = {1: '1', 2: '2', 3: '3', 4: '4', 5: '5', 6: '6', 7: '7', 8: '8', 9: '9', 10: '10'}
+
+                testParameters = {1: 'testExecutionId', 2: 'testOwner', 3: 'testStartTime', 4: 'testEndTime',
+                                  5: 'testDuration'}
+
+                sut_Parameters = {1: 'dataNodeCount', 2: 'masterNodeCount', 3: 'dataNodeInstanceType',
+                                  4: 'masterNodeInstanceType', 5: 'elasticsearchDistributionType', 6: 'aes',
+                                  7: 'clusterName', 8: 'awsAccount', 9: 'endpoint',
+                                  10: 'region'}
+                work_config = {1: 'dataset', 2: 'warmupIterations', 3: 'testIterations'}
+
+                def testResults_OS(arg1):
+                    if arg1 == 'index':
+                        arg1 = 'index'
+                    else:
+                        arg1 = 'query'
+
+                    for i in range(1, 11):
+                        if i == 4 or i == 5 or i == 6:
```

Comment:
`Code Smell: In a loop you change behavior based on index`
This code is fragile as the ordering of the incoming data shapes your behavior, if a field is added not only do you need to update this line, but line 68 as well.